cypress/integration/ProcessorTest.ts

```diff
@@ -0,0 +1,449 @@
+describe("Processor Test", () => {
+  /**
+   * This test suite verifies that the user can create all types of processors.
+   * Because it is running with mocked data we have to assert all data immediately (no reload of the page).
+   */
+  describe("Create a Processor", () => {
+    beforeEach(() => {
+      cy.visit("/instance/830c8f0d-c677-492f-8d7e-0f81893fbba6");
+      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
+        "not.exist"
+      );
+    });
+
+    it("Sink processor", () => {
+      const processorName: string = "Sink processor";
+      const action = [
+        "Send to Slack",
+        "dev-action",
+        "https://test.app.com/item",
+      ];
+      const filters = [
+        ["data.name", "processor.StringEquals", "John"],
+        ["data.surname", "processor.StringEquals", "White"],
+      ];
+
+      cy.ouiaId("create-processor", "PF4/Button").click();
+      cy.ouiaId("Create processor", "breadcrumb-item").should("be.visible");
+      cy.ouiaId("sink", "Tile").should("be.visible").click();
+      cy.ouiaId("processor-name", "PF4/TextInput")
+        .should("be.visible")
+        .type(processorName);
+
+      //Filters
+      cy.ouiaId("add-filter", "PF4/Button").should("be.visible").click();
+
+      cy.ouiaId("item-0").within(() => {
+        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[0][0]);
+        cy.ouiaId("filter-type", "PF4/FormSelect").select("String equals");
+      });
+      cy.ouiaId("item-0") //The filter-value was detached from DOM and we need to find the context again.
+        .ouiaId("filter-value", "PF4/TextInput")
+        .type(filters[0][2]);
+      cy.ouiaId("item-1").within(() => {
+        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[1][0]);
+        cy.ouiaId("filter-type", "PF4/FormSelect").select("String equals");
```

Comment:
related to my previous question , shouldn't we select `filters[1][1]` or sth similar?